projects/nuvoton/numaker_iot_m487_wifi/uvision/aws_demos_eth/aws_demos_eth.uvproj

```diff
@@ -1608,6 +1608,26 @@
 						</File>
 					</Files>
 				</Group>
+				<Group>
+					<GroupName>libraries/coreJSON/source/</GroupName>
+					<Files>
+						<File>
+							<FileName>core_json.c</FileName>
+							<FileType>1</FileType>
+							<FilePath>../../../../../libraries/coreJSON/source/core_json.c</FilePath>
+						</File>
+					</Files>
+				</Group>
+				<Group>
+					<GroupName>libraries/device-shadow-for-aws-iot-embedded-sdk/source/</GroupName>
+					<Files>
+						<File>
+							<FileName>shadow.c</FileName>
+							<FileType>1</FileType>
+							<FilePath>../../../../../libraries/device-shadow-for-aws-iot-embedded-sdk/source/shadow.c</FilePath>
+						</File>
+					</Files>
+				</Group>
 				<Group>
```

Comment:
Please update `<IncludePath>` tag, otherwise the build won't be able to find the include headers when the shadow demo is enabled.

Please add `../../../../../libraries/coreJSON/source/include;../../../../../libraries/device-shadow-for-aws-iot-embedded-sdk/source/include` to end of the list

Reply:
thanks.